seed/object_types.go

```diff
@@ -37,11 +38,13 @@ func seedObjectTypes(node universe.Node) error {
 						miroPluginID,
 					},
 				},
-				Private:           utils.GetPTR(false),
-				Visible:           utils.GetPTR(entry.UI2DObjectVisibleType),
-				DefaultTiles:      []any{},
-				FrameTemplates:    map[string]any{},
-				AllowedSubObjects: []umid.UMID{},
+				Private:        utils.GetPTR(false),
+				Visible:        utils.GetPTR(entry.UI2DObjectVisibleType),
+				DefaultTiles:   []any{},
+				FrameTemplates: map[string]any{},
+				AllowedChildren: []string{
+					"a41ee21e-6c56-41b3-81a9-1c86578b6b3c",
```

Comment:
the others you gave a nice comment about what it is... now you have to do it here too :)